flume/CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,5 @@
 * Encourage the usage of FQDNs regarding HDFS namenodes when using Kerberos (#309)
 * Fixed the RoundRoundChannelSelector, now it works for multiple storages (#298)
 * Allow for infinite TTL (#236)
+* Added OrionTestSink (#307)
+* Fixed OrionMySQLSink logger (now it logs as OrionMySQLSink, not as OrionHDFSSink)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/26de0cb95ca537a027191777742cbad2b2e3aae7